static/src/javascripts/bootstraps/common.js

```diff
@@ -227,6 +227,25 @@ define([
                 });
             },
 
+            // Adds a global window:throttledScroll event to mediator, which throttles
+            // scroll events until there's a spare animationFrame.
+            // Callbacks of all listeners to window:throttledScroll are run in the
+            // same animationFrame, meaning they can all perform DOM reads for free
+            // (after the first one that needs layout triggers it).
+            // However, this means it's VITAL that all writes in callbacks are delegated to fastdom
+            throttledScrollEvent: function () {
+                var running = false;
+                window.addEventListener('scroll', function () {
+                    if (!running) {
+                        running = true;
+                        requestAnimationFrame(function () {
```

Comment:
According to caniuse this is not supported in i.e. 9. Do we polyfill `requestAnimationFrame`?

Also have you considered `defer` in Fast Dom https://github.com/wilsonpage/fastdom#fastdomdeferframes-callback-context